python_onnxifi/onnxifi_backend.py

```diff
@@ -0,0 +1,53 @@
+from build.python_onnxifi import *
+import onnx
+from onnx import (NodeProto,
+                  ModelProto)
+
+class OnnxifiBackendRep(object):
+    #Constructor never be used - underlying C++ object created behind the scenes
+    #in OnnxifiBacken::prepare
+    def run(self, inputs, **kwargs):  # type: (Any, **Any) -> Tuple[Any, ...]
+        self.backend_rep_.run(inputs, **kwargs)
+
+
+class OnnxifiBackend(object):
```

Comment:
Let's make OnnxifiBackend a subclass of Backend in https://github.com/onnx/onnx/blob/master/onnx/backend/base.py

Reply:
@bddppq and I agreed to make functions in the OnnxifiBackend member functions as opposed to static (for clearer resource management/garbage collection). Would it be awkward/confusing to have an inheritance relationship given that base.py uses class methods instead?